### ISSUE TITLE
Advanced feature issues when creating a Lesson

### ISSUE THREAD

**Issue by jenjac24** [codeforamerica/bizfriendly-web#188]
1. When you click on advanced features, then select attribute equals expected values, a feedback panel appears but the next button does not appear to do anything when you click it.

2.Sometimes when you click on advanced options and select a first option and then try to select a second option, the second option(What kind of item?) does not have any options.

**Comment by ondrae** [comment]
You're right, the whole Advanced section doesn't work unless you know exactly what you're doing.

BizFriend.ly has a really cool interactive lesson feature, where it can tell when you've successfully done something on a third party service, such as checkin on Foursquare. The Advanced section was a very basic attempt at putting a UI on that part. I want to leave it in, so I can continue to demo the features to people.
